### DISCUSSION TITLE
Attaching javaagent to multiple JVMs

### DISCUSSION THREAD

**Discussion by sarkisjad** [open-telemetry/opentelemetry-java-instrumentation#11198] [Q&A]
I am attaching my opentelemetry javaagent to multiple JVMs.
The javaagent auto instruments [JMX](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics/javaagent) and they are sent to an otlp receiver. 
My problem is that I only get auto instrumented metrics from one or two JVMs and not all the JVMs. 
Is there a specific property I should add to fix this problem? If not, how could I debug this problem further?

**Comment by laurit** [comment]
If you aren't able to figure this out on your own then look at the processes where metrics aren't collected and try to create a minimal sample application that reproduces the issue.
